config/initializers/devise.rb

```diff
@@ -251,7 +251,8 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-
+  config.omniauth :facebook, "100780687136647", "c20b6ad31d880322a509ecee78f44fe5",
```

Comment:
you included figaro - please use ENV variables here instead of hard coding the key